lib/kitchen/provisioner/chef_base.rb

```diff
@@ -47,18 +48,6 @@ class ChefBase < Base
         providers/**/* recipes/**/* resources/**/* templates/**/*
       ].join(",")
 
-      default_config :chef_metadata_url do |provisioner|
-        provisioner.default_windows_chef_metadata_url if provisioner.windows_os?
-      end
-
-      default_config :chef_omnibus_root do |provisioner|
```

Comment:
If a user needs to override their omnibus install location, how do they do it now?  Is this supported anymore?